docs/ClusterSetup.md

```diff
@@ -221,6 +224,7 @@ zookeeperServers=zk1.us-west.example.com:2181,zk2.us-west.example.com:2181,zk3.u
 globalZookeeperServers=zk1.us-west.example.com:2184,zk2.us-west.example.com:2184,zk3.us-west.example.com:2184
 
 clusterName=us-west
+statusFilePath=broker-status-file
```

Comment:
I just tried again to double-check. The status file settings that are in the `conf/broker.conf` file are "fine", in the sense that they don't prevent the broker from starting correctly. 

About the `statusFilePath` setting, this is used to control whether the broker should be considered as "in-rotation" or "out-of-rotation" from an external monitor process. I'm not sure how useful would this be outside of Yahoo though.